lib/datadog_api_client/v1/models/metrics_query_metadata.rb

```diff
@@ -95,7 +95,7 @@ def self.openapi_types
         :'interval' => :'Integer',
         :'length' => :'Integer',
         :'metric' => :'String',
-        :'pointlist' => :'Array<Array>',
+        :'pointlist' => :'Array<Array<Float>>',
```

Comment:
This is generated code. We need to fix it in the code generator.

Reply:
Thanks for the fast reply.... Do you mind help me where I can find the code generator?

Reply:
We need to trigger the code generation manually, but it looks more like issue with `openapi-generator` itself since https://github.com/DataDog/datadog-api-client-ruby/blob/master/.generator/templates/partial_model_generic.mustache#L62 looks fine.